server/modules/MainServer.js

```diff
@@ -23,6 +24,17 @@ let serverInstance;
 
 const APP_VERSION = require('config').get('APP_VERSION');
 
+function usernameMiddleware(req, res, next) {
+  try {
```

Comment:
Hi @duncanhall. Good point. I've factored logging configuration and usernameMiddleware into separate modules.